modules/claims_api/lib/evss_service/base.rb

```diff
@@ -29,7 +29,7 @@ def submit(claim, data, async = true) # rubocop:disable Style/OptionalBooleanPar
           resp
         rescue => e
           detail = e.respond_to?(:original_body) ? e.original_body : e
-          log_outcome_for_claims_api('validate', 'error', detail, claim)
+          log_outcome_for_claims_api('submit', 'error', detail, claim)
```

Comment:
this is just a typo from another PR.. copied from the log line for validate, we had "validate" here instead of "submit" which is the action. Made debugging the logs just a bit more difficult but nothing pressing in this change.